src/firejail/run_symlink.c

```diff
@@ -76,6 +76,10 @@ void run_symlink(int argc, char **argv, int run_as_is) {
 		a[i + 2] = argv[i + 1];
 	}
 	a[i + 2] = NULL;
+	if (env_get("LD_PRELOAD") != NULL)
+	{
+		fprintf(stderr, "run_symlink: LD_PRELOAD is: '%s'\n", env_get("LD_PRELOAD"));
+	}
```

Comment:
```suggestion
	if (env_get("LD_PRELOAD") != NULL)
		fprintf(stderr, "run_symlink: LD_PRELOAD is: '%s'\n", env_get("LD_PRELOAD"));
```

Follow the surrounding `if` style.

Reply:
Sure! Done.